recipes/zoneinfo/tzdata-location.patch

```diff
@@ -0,0 +1,23 @@
+index 9baaf6b..51cf452 100644
+--- a/src/backports/zoneinfo/_tzpath.py
++++ b/src/backports/zoneinfo/_tzpath.py
+@@ -22,17 +22,8 @@ def reset_tzpath(to=None):
+         env_var = os.environ.get("PYTHONTZPATH", None)
+         if env_var is not None:
+             base_tzpath = _parse_python_tzpath(env_var)
+-        elif sys.platform != "win32":
+-            base_tzpath = [
+-                "/usr/share/zoneinfo",
+-                "/usr/lib/zoneinfo",
+-                "/usr/share/lib/zoneinfo",
+-                "/etc/zoneinfo",
+-            ]
+-
+-            base_tzpath.sort(key=lambda x: not os.path.exists(x))
+         else:
+-            base_tzpath = ()
++            base_tzpath = [os.path.join(sys.prefix, "share", "zoneinfo")]
```

Comment:
I wonder if it'd make a difference to use `sys.base_prefix` instead of `sys.prefix` -- but my gut feeling is that either should work the same since no one would install the Conda package inside a virtualenv.
(^-- Just wording out my thoughts; the patch looks good to me either way!)

Reply:
I didn't even know about `base_prefix`. Doesn't hurt to be defensive I suppose...

Reply:
I would lie if I'd say *I really knew* about it -- but an itch far far back in my head reminded me there was something XD.